e2e/drift/drift_test.go

```diff
@@ -18,6 +19,9 @@ var _ = Describe("Drift", func() {
 		k          kubectl.Command
 		namespace  string
 		bundleName string
+
+		interval = 2 * time.Second
+		duration = 30 * time.Second
```

Comment:
This feels like a long time, on the other hand we know that end-to-end tests can be slow on Github runners.

Reply:
I know, but I just copied it from somewhere else. I'm fact, this is not a timeout, but used for a `Consistently()` statement. It's the time that during which it will continue checking if the condition is still true.
Please let me know if you still want to change this to 10 or 15 seconds.

Reply:
No worries, let's leave it at this for now; what I meant is that precisely because of slowness on Github runners, it makes sense for `Consistently` to run over a longer duration, just to be on the safe side.